docs/TypeScript.md

```diff
@@ -123,9 +123,9 @@ js_library(
           "node_modules/**/* *",
         ],
     ),
-    # Provide ExternalNpmPackageInfo which is used by downstream rules
-    # that use these npm dependencies
-    external_npm_package = True,
+    # Special value to provide ExternalNpmPackageInfo which is used by downstream
+    # rules that use these npm dependencies
+    package_name = "$node_modules$",
```

Comment:
Switching to a special token on `package_name` instead of using `external_npm_package = True`.
Reason is I got rid of the "internal only" `external_package_path` as package_path is used for both 3p and 1p deps so it seems cleaner to also get right of the "internal only" `external_npm_package` as a special package_name could be used instead.

Maybe too confusing to make this switch?